### PR TITLE
uniextract2: Add version 2.0.0-rc.2

### PR DESCRIPTION
**Pull request by issaclin32**
Official homepage for UniExtract2: https://github.com/Bioruebe/UniExtract2
Please let me know if there are any problems. Thanks.

### CODE REVIEW

File: bucket/uniextract2.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    ],
+    "shortcuts": [
+        [
+            "UniExtract.exe",
+            "Universal Extractor"
+        ]
+    ],
+    "checkver": {
+        "url": "https://github.com/Bioruebe/UniExtract2/releases",
+        "regex": "\\/Bioruebe\\/UniExtract2\\/releases\\/download\\/v(?<version>[\\w.-]+)\\/(?<filename>\\w+\\.zip)"
```
Comment (Ash258) [inline bucket/uniextract2.json]: No need to be so strict.

```suggestion
        "regex": "download/v([\\w.-]+)/(?<filename>\\w+\\.zip)"
```

Comment (Ash258) [reply bucket/uniextract2.json]: A

File: bucket/uniextract2.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+    "version": "2.0.0-rc.2",
+    "description": "A program designed to decompress and extract files from any type of archive or installer.",
```
Comment (Ash258) [inline bucket/uniextract2.json]:
```suggestion
    "description": "Decompress and extract files from any type of archive or installer.",
```

**Comment by rasa** [comment]
In general, do we want to allow `-rc` builds? If so, perhaps we should modify scoop, instead of processing them manually?

**Comment by niheaven** [comment]
For UniExtract2, we need RC build, since its stable version (maybe 2.0.0) is still not within the foreseeable future.

**Pull request merged**